packages/cli/cli/src/persistence/getPathToOutputDirectoriesFile.ts

```diff
@@ -0,0 +1,13 @@
+import { AbsoluteFilePath, join, RelativeFilePath } from "@fern-api/fs-utils";
+import { homedir } from "os";
+
+const APPROVED_OUTPUT_DIRECTORIES_FILENAME = "approved-output-directories";
+const LOCAL_STORAGE_FOLDER = process.env.LOCAL_STORAGE_FOLDER ?? ".fern";
```

Comment:
nit: there is a `consts` file that we can use for some of this 

https://github.com/fern-api/fern/blob/main/packages/cli/cli/src/constants.ts